pkg/policies/branch/branch_test.go

```diff
@@ -651,6 +651,8 @@ func TestCheck(t *testing.T) {
 
 func TestFix(t *testing.T) {
 	flse := false
+	var appid int64
+	appid = 123
```

Comment:
Nit: use single line definition: `var appid int64 = 123`

Reply:
Ah cool! Looks like there are already some helper funcs to get pointers in the github library, switched to those.